### ISSUE TITLE
Limit playback duration

### ISSUE THREAD

**Issue by bruferrari** [brianwernick/ExoMedia#430]
Is there a way to limit the playback duration by exomedia? I want to limit a video playback that has 80 seconds to 60 seconds, i don't care about the lost video parts, just want to limit it, is there a way to achieve it with this lib?

**Comment by brianwernick** [comment]
There's nothing built in that would directly support this in it's entirety, however there are a few options:
  1. Monitor the current progress and just stop at the appropriate position (in your example 60 seconds), and disallow seeking beyond that
  2. Build a custom `RenderBuilder` (assuming 3.x, `MediaSource` for 4.x) or HttpDataSource that will correctly limit the playback by not loading beyond that point

In either case you can also override the duration of the media (with `overrideDuration()`) which will present the UI as if that is the max value, however this alone won't stop playback from moving beyond the specified value

**Comment by bruferrari** [comment]
Thanks for your tips, I managed it by the first approach by using `Timer().scheduleAtFixedRate` to monitor when the video video current position is equals to 60 seconds, I don't know if it's the best solution but for now it solved my problem. I've tried `overrideDuration()` before, didn't work like you said, it just modifies the UI and not the playback itself.

Thanks again and congrats for this awesome lib by the way.
